tests/test_ari_text.py

```diff
@@ -875,7 +875,7 @@ def test_ari_text_decode_lit_typed_cbor(self):
             ("ari:/CBOR/h''", b""),
             ("ari:/CBOR/h'A164746573748203F94480'", b'\xa1dtest\x82\x03\xf9D\x80'),
             ("ari:/CBOR/h'0064746573748203F94480'", b'\x00dtest\x82\x03\xf9D\x80'),
-            # FIXME: ("ari:/CBOR/h'A1%2064%2074%2065%2073%2074%2082%2003%20F9%2044%20%2080'", b"A164746573748203F94480")
+            ("ari:/CBOR/h'A1%2064%2074%2065%2073%2074%2082%2003%20F9%2044%20%2080'", b"A164746573748203F94480")
```

Comment:
There was a typo in the new line, but it's easier to follow the test if the expected values are themselves decoded from hex.
```suggestion
("ari:/CBOR/h'A164746573748203F94480'", bytes.fromhex('A164746573748203F94480')),
("ari:/CBOR/h'0064746573748203F94480'", bytes.fromhex('0064746573748203F94480')),
            ("ari:/CBOR/h'A1%2064%2074%2065%2073%2074%2082%2003%20F9%2044%20%2080'", bytes.fromhex("A164746573748203F94480"))
```